native-sql-engine/core/src/main/scala/com/intel/oap/extension/columnar/ColumnarGuardRule.scala

```diff
@@ -257,7 +260,7 @@ case class ColumnarGuardRule() extends Rule[SparkPlan] {
       case p if !supportCodegen(p) =>
         // insert row guard them recursively
         p.withNewChildren(p.children.map(insertRowGuardOrNot))
-      case p: CustomShuffleReaderExec =>
```

Comment:
TODO: change to if statement and ask shim layer to judge where is CustomShuffleReaderExec type (for spark3.1) or AQEShuffleReadExec type (for spark3.2).